blockchain/validate.go

```diff
@@ -1133,6 +1217,16 @@ func (b *BlockChain) checkConnectBlock(node *blockNode, block *btcutil.Block, vi
 		scriptFlags |= txscript.ScriptVerifyCheckLockTimeVerify
 	}
 
+	// Enforce CHECKSEQUENCEVERIFY during all block validation checks once
+	// the soft-fork deployment is fully active.
+	csvState, err := b.thresholdHoldForDeployment(chaincfg.DeploymentCSV)
```

Comment:
And this would need to be:

```Go
csvState, err := b.thresholdHoldForDeployment(prevNode, chaincfg.DeploymentCSV)
```

so it is from the point of view of the block being checked instead of the main chain.